### ISSUE TITLE
How to acces script.js directly in heroku?

### ISSUE THREAD

**Issue by ghost** [Apress/building-chatbots-with-python#7]
![image](https://user-images.githubusercontent.com/16435896/68073663-60764f00-fdcd-11e9-8b54-3f7eb563def9.png)

I change script.js path to /assets/js/script.js in index.html,
but then open index.html without chatbot both in local and heroku?

![image](https://user-images.githubusercontent.com/16435896/68073693-a92e0800-fdcd-11e9-9b7d-771408d58ec6.png)

**Comment by ghost** [comment]
I have upload js directly to aws also, but bot seams not work right , maybe model bug?
below is my chatbot demo
https://horoscopebot1001.herokuapp.com/webhooks/rasa/chat

**Comment by sumit12dec** [comment]
I can see your chatbot coming up in the UI and interacting with the API to take requests and return responses as well. 

Can you tell me if you are facing the same issue or different?

Let me know if you are still having the issue.
<img width="1424" alt="Screen Shot 2019-11-03 at 11 59 21 AM" src="https://user-images.githubusercontent.com/1473477/68081281-8df4e400-fe31-11e9-93c5-1624ab994887.png">

**Comment by ghost** [comment]
yes, I directly upload single script.js file to a aws bucket, when  I answer “capricorn", it died without response, maybe the model is not right, I will check it later


![image](https://user-images.githubusercontent.com/16435896/68081721-85120b80-fe4d-11e9-9b86-54d446c02e9a.png)

**Comment by sumit12dec** [comment]
Yeah.. Let us know when you resolve the issue. Feel free to ask again if you need anything.

It'll be very kind of you if you could put up your review on Amazon.
https://www.amazon.com/Building-Chatbots-Python-Language-Processing/dp/1484240952
